src/iqm/iqm_client/iqm_client.py

```diff
@@ -854,6 +858,82 @@ def get_quantum_architecture(self, *, timeout_secs: float = REQUESTS_TIMEOUT) ->
         self._architecture = qa
         return qa
 
+    def get_quality_metric_set(self, *, timeout_secs: float = REQUESTS_TIMEOUT) -> QualityMetricSet:
+        """Retrieve the latest quality metric set from the server using the V1 API 
+        (Cocos circuits execution and Resonance) architecture.
+
+        Caches the result and returns the same result on later invocations.
+
+        Args:
+            timeout_secs: network request timeout
+
+        Returns:
+            quality metrics
+
+        Raises:
+            QualityMetricsRetrievalError: IQM server specific exceptions
+            ClientAuthenticationError: if no valid authentication is provided
+            HTTPException: HTTP exceptions
+        """
+
+        result = requests.get(
+            self._api.url(APIEndpoint.QUALITY_METRICS_LATEST),
+            headers=self._default_headers(),
+            timeout=timeout_secs,
+        )
+
+        self._check_not_found_error(result)
+        self._check_authentication_errors(result)
+        result.raise_for_status()
+
+        try:
+            qm = QualityMetricSet(**result.json())
+        except (json.decoder.JSONDecodeError, KeyError) as e:
+            raise QualityMetricSetRetrievalError(f'Invalid response: {result.text}, {e}') from e
+
+        return qm
+
+    def get_calibration_set(
+        self, calibration_set_id: Optional[UUID] = None, *, timeout_secs: float = REQUESTS_TIMEOUT
+    ) -> CalibrationSet:
+        """Retrieve calibration set from the server using the V1 API (Cocos circuits execution and Resonance)
```

Comment:
```suggestion
        """Retrieve calibration set from the server using the V1 API (Cocos circuits execution and Resonance).
```
First line must end in period, I think Ruff checks this.